catkin_home/src/navigation/base_control/scripts/encoders.py

```diff
@@ -1,58 +1,57 @@
 #!/usr/bin/env python
-'''
-    Script that simulate physical robot action of sending encoders ticks.
-'''
 import rospy
+from geometry_msgs.msg import Twist
 from base_control.msg import StampedEncoders
 
-def run_encoders_simulation():
-    '''
-        Main function that simulate the action of sending encoders ticks.
-    '''
-    # Starts a new node.
+def move():
+    # Starts a new node
     rospy.init_node('encoders_simulation', anonymous=True)
-    rate = rospy.Rate(4) # ROS Rate at 4Hz.
-    front_publisher = rospy.Publisher('/base_control/front/encoders',StampedEncoders,queue_size=10)
-    back_publisher = rospy.Publisher('/base_control/back/encoders',StampedEncoders,queue_size=10)
+    rate = rospy.Rate(4) # ROS Rate at 4Hz
+    front_publisher = rospy.Publisher('/base_control/front/encoders', StampedEncoders, queue_size=10)
+    back_publisher = rospy.Publisher('/base_control/back/encoders', StampedEncoders, queue_size=10)
+    
     rospy.loginfo("Node Initiated")
-
-    # Initialize Encoder Message.
-    enc_msg_front = StampedEncoders()
-    enc_msg_back  = StampedEncoders()
-    enc_msg_front.encoders.time_delta = 0.04
-    enc_msg_back.encoders.time_delta = 0.04
-
-    pulses = 100
-    # Direction:       0      ,180    ,90    ,270    ,45    ,225    ,135   ,315    ,Left  ,Right
-    enc_front_left  = [pulses ,-pulses,pulses,-pulses,pulses,-pulses,0     ,0      ,-pulses,pulses ]
-    enc_back_left   = [-pulses,pulses ,pulses,-pulses,0     ,0      ,pulses,-pulses,-pulses,pulses ]
-    enc_front_right = [-pulses,pulses ,pulses,-pulses,0     ,0      ,pulses,-pulses,pulses ,-pulses]
-    enc_back_right  = [pulses ,-pulses,pulses,-pulses,pulses,-pulses,0     ,0      ,pulses ,-pulses]
-
-    last_publish = rospy.Time.now().to_sec()
-    last_change = rospy.Time.now().to_sec()
-    index_direction = 0
-
+    
+    #Initialize Twist Message
+    enc_msgFront = StampedEncoders()
+    enc_msgBack  = StampedEncoders()
+    enc_msgFront.encoders.time_delta = 0.005
+    enc_msgBack.encoders.time_delta = 0.005
+
+    forward  = 10000
+    backward = -10000
+    stop     = 0 
```

Comment:
The old values were simulating the actual encoders. Why the need of changing these values?